lib/utils.js

```diff
@@ -191,13 +191,22 @@ exports.getLatestRelease = function () {
 // Try to match a request to a template, for example a request for /test
 // would look for /app/views/test.html
 // or /app/views/text/index.html
-exports.matchRoutes = function (req, res) {
+exports.matchRoutes = function (req, res, next) {
   var path = (req.params[0])
   res.render(path, function (err, html) {
     if (err) {
+      // We don't want to error on every 'template not found', as it might not
+      // be found as test.html, but then found as test/index.html. If neither is
+      // found, we log a single error at the end (in server.js)
+      if (!err.message.startsWith('template not found')) {
+        console.error(err.message)
```

Comment:
we could try and pass this to `next` - but we'd need to send both this error and the next one (line 207)